tests/schema_registry/test_avro_serializer.py

```diff
@@ -74,6 +74,29 @@ def test_avro_serializer_config_auto_register_schemas_false(mock_schema_registry
     assert test_client.counter['POST'].get('/subjects/{}'.format(subject)) == 1
 
 
+def test_avro_serializer_config_use_latest_version(mock_schema_registry):
+    """
+    Ensures auto.register.schemas=False does not register schema
+    """
+    conf = {'url': TEST_URL}
+    test_client = mock_schema_registry(conf)
+    topic = "test-use-latest-version"
+    subject = topic + '-key'
+
+    test_serializer = AvroSerializer(test_client, 'string',
+                                     conf={'auto.register.schemas': False, 'use.latest.version': True})
+
+    test_serializer("test",
+                    SerializationContext("test-use-latest-version",
+                                         MessageField.KEY))
+
+    register_count = test_client.counter['POST'].get('/subjects/{}/versions'
+                                                     .format(subject), 0)
+    assert register_count == 0
+    # Ensure latest was requested
+    assert test_client.counter['GET'].get('/subjects/{}/versions/latest'.format(subject)) == 1
```

Comment:
there should really be more test coverage of this stuff.
but i'm ok with the effort/benefit tradeoff.